frontend/Jenkinsfile-frontend

```diff
@@ -62,18 +62,21 @@ pipeline {
                 }
             }
             steps {
-                withCredentials([ 
-                    string(credentialsId: 'KC_FRONTEND_CI_API_URL', variable: 'API_URL'),
-                    string(credentialsId: 'KC_FRONTEND_CI_ORCID_CLIENT_ID', variable: 'ORCID_CLIENT_ID'),
-                    string(credentialsId: 'KC_FRONTEND_CI_ORCID_CLIENT_SECRET', variable: 'ORCID_CLIENT_SECRET'),
-                    string(credentialsId: 'KC_FRONTEND_CI_FRONTEND_URL', variable: 'FRONTEND_URL')
-                ]) {    
-                    script {
-                        docker.build("${env.DOCKER_REPO_NAME}", "--build-arg API_URL=${API_URL} --build-arg ORCID_CLIENT_ID=${ORCID_CLIENT_ID} --build-arg ORCID_CLIENT_SECRET=${ORCID_CLIENT_SECRET} --build-arg FRONTEND_URL=${FRONTEND_URL} --no-cache ./frontend")
-                        docker.withRegistry('https://853771734544.dkr.ecr.us-east-1.amazonaws.com', 'ecr:us-east-1:aws-ifx-deploy') {
-                            docker.image(env.DOCKER_REPO_NAME).push("${BUILD_VERSION}")
-                        }  
-                    }
+                configFileProvider([
+                    configFile(fileId: 'frontend-build-prepare.sh', targetLocation: 'frontend/frontend-build-prepare.sh')
+                ]){
+                   dir("./frontend") {    
+                        script {
+                            sh '''
+                            /bin/bash frontend-build-prepare.sh
+                            docker build --no-cache -f ./Dockerfile --build-arg API_URL=${API_URL} --build-arg ORCID_CLIENT_ID=${ORCID_CLIENT_ID} --build-arg ORCID_CLIENT_SECRET=${ORCID_CLIENT_SECRET} --build-arg FRONTEND_URL=${FRONTEND_URL} -t 853771734544.dkr.ecr.us-east-1.amazonaws.com/translator-cdskp-collaboratory-frontend .
```

Comment:
Please use the defined DOCKER_REPO_NAME env in this command. and same changes to be applied to the docker build commands later in this Jenkinsfile.